src/components/RecipesList/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, memo } from "react";
+import { Grid } from "@mui/material";
+import { useQuery } from "react-query";
+import { IRecipe } from "../../interfaces/Recipe";
+import RecipesItem from "../RecipeItem";
+import RecipeSkeleton from "../RecipeItem/Skeleton";
+import RecipesAPI from "../../api/Recipes";
+
+interface IList {
+  queryString: string;
+}
+
+const RecipesList: FC<IList> = ({ queryString }) => {
+  const { isLoading, error, data } = useQuery<{ results: IRecipe[] }, Error>(
+    ["RecipesListComplexSearch", queryString],
+    () => RecipesAPI.getComplexQueryRecipes(queryString)
+  );
+
+  if (isLoading)
+    return (
+      <Grid container spacing={3}>
+        {[1, 2, 3, 4].map((i) => (
```

Comment:
what is this used for? does it stay the same size FOREVER????

Reply:
Yes it's stay the same size because it is a skeleton to show the user while loading the data